lib/src/select/Select.test.js

```diff
@@ -351,6 +406,27 @@ describe("Select component tests", () => {
     fireEvent.keyDown(select, { key: "ArrowUp", code: "ArrowUp", keyCode: 38, charCode: 38 });
     expect(select.getAttribute("aria-activedescendant")).toBe("option-0");
   });
+  test("Non-Grouped Options - Filtering options never affects the optional item until there are no coincidence", () => {
+    const { getByRole, getAllByRole, getByText, queryByText, container } = render(
+      <DxcSelect
+        label="test-select-label"
+        placeholder="Placeholder example"
+        options={single_options}
+        optional
+        searchable
+      />
+    );
+    const select = getByRole("combobox");
+    const searchInput = container.querySelectorAll("input")[1];
+
+    userEvent.click(select);
+    userEvent.type(searchInput, "1");
+    expect(getByText("Placeholder example")).toBeTruthy();
```

Comment:
The optional item has the placeholder text as its label. I'm checking its presence in this test when you filter the list since this action shouldn't affect it.

Reply:
Now i get it!! 🤦🏼‍♂️